py/server/deephaven/table_listener.py

```diff
@@ -347,7 +358,8 @@ def stop(self) -> None:
 
         if not self.started:
             return
-        self.t.j_table.removeUpdateListener(self.listener_adapter)
+        with update_graph.auto_locking_ctx(self.t):
+            self.t.j_table.removeUpdateListener(self.listener_adapter)
```

Comment:
You don't need a lock to remove.
```suggestion
        self.t.j_table.removeUpdateListener(self.listener_adapter)
```